### ISSUE TITLE
Install Minikube in Windows 10 professional

### ISSUE THREAD

**Issue by sudhirj8** [sebgoa/oreilly-kubernetes#8]
I am using Windows 10 professional, 
Downloaded minikube using the steps in the link below
https://blogs.msdn.microsoft.com/wasimbloch/2017/01/23/setting-up-kubernetes-on-windows10-laptop-with-minikube/

But received below error
Starting local Kubernetes v1.12.4 cluster...
Starting VM...
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive ( Hyper-V\Get-VM minikube ).state
[stdout =====>] : Running

[stderr =====>] :
Waiting for SSH to be available...
Getting to WaitForSSH function...
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive ( Hyper-V\Get-VM minikube ).state
[stdout =====>] : Running

[stderr =====>] :
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive (( Hyper-V\Get-VM minikube ).networkadapters[0]).ipaddresses[0]
[stdout =====>] :
[stderr =====>] :
Error getting ssh command 'exit 0' : IP not found
Getting to WaitForSSH function...
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive ( Hyper-V\Get-VM minikube ).state
[stdout =====>] : Running

**Comment by sebgoa** [comment]
Did you follow the step about the "virtual switch manager" ? that's critical for minikube to get an IP you can reach even from the host itself.

**Comment by sudhirj8** [comment]
Yes, I followed the steps for Virtual switch manager as given on the site
